src/azure-cli-core/azure/cli/core/extension/operations.py

```diff
@@ -111,7 +111,7 @@ def _add_whl_ext(cli_ctx, source, ext_sha256=None, pip_extra_index_urls=None, pi
         raise CLIError('Unable to determine extension name from {}. Is the file name correct?'.format(source))
     if extension_exists(extension_name, ext_type=WheelExtension):
         raise CLIError('The extension {} already exists.'.format(extension_name))
-    if extension_name == 'rdbms-connect':
+    if extension_name == 'rdbms-connect' or extension_name == 'serviceconnector-passwordless':
```

Comment:
how about using a more common name, for example, `serviceconnector-extension`, in case we may have more features in the extension

Reply:
The extension `serviceconnector-passwordless` requires users to install many dependencies in order to connect postgresql/mysql/sql. Users may have concerns of installing them if they don't want to connect databases. I think we should separate passwordless command from other Service Connector commands and leave `serviceconnector-extension` for future real serviceconnector-extension